### PR TITLE
Update ODAC requirements to avoid customer confusion.

### PR DESCRIPTION
**Pull request by rufengli**
Update ODAC requirements to avoid customer confusion.

### CODE REVIEW

**Comment by PRMerger17** [comment]
@rufengli : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: powerbi-docs/connect-data/desktop-connect-oracle-database.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ To connect to an Oracle database with Power BI Desktop, the correct Oracle clien
 
 Supported Oracle versions: 
 - Oracle Server 9 and later
-- Oracle Data Access Client (ODAC) software 11.2 and later
+- Oracle Data Access Client (ODAC) software 11.2 or greater
 
-Before you can connect to an Oracle database using Power BI, you need to install the Oracle client software v8.1.7 or greater on your computer. To install the 32-bit Oracle client software, go to [32-bit Oracle Data Access Components (ODAC) with Oracle Developer Tools for Visual Studio (12.1.0.2.4)](https://www.oracle.com/technetwork/topics/dotnet/utilsoft-086879.html). To install the 64-bit Oracle client, go to [64-bit ODAC 12c Release 4 (12.1.0.2.4) Xcopy for Windows x64](https://www.oracle.com/technetwork/database/windows/downloads/index-090165.html).
```
Comment (katwan) [inline powerbi-docs/connect-data/desktop-connect-oracle-database.md:20]: To keep it consistent, should we change "To install the 32-bit Oracle client software" to "To install the 32-bit ODAC software"?

**Comment by PRMerger6** [comment]
@rufengli : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by davidiseminger** [comment]
Looks good -thanks for the contribution!

**Pull request merged**